### ISSUE TITLE
[Request] allow generation of plugin goal pages under a child

### ISSUE THREAD

**Issue by wattazoum** [bsorrentino/maven-confluence-plugin#118]
Hi, 

Currently AFAIK , the goal pages are generated under the HomePage. This mess up  with the definition that we have done in the site.xml. 
We should be able to choose under which page we want the plugins goals to be.

**Comment by wattazoum** [comment]
I can suggest something like 
```xml
  <xs:complexType name="page">
    <xs:complexContent>
      <xs:extension base="tns:source">
        <xs:sequence>
          <xs:element name="attachment" type="tns:attachment" minOccurs="0" maxOccurs="unbounded"/>
          <xs:element name="child" type="tns:page" minOccurs="0" maxOccurs="unbounded"/>
          <xs:element name="label" type="xs:string" minOccurs="0" maxOccurs="unbounded"/>
          <xs:element name="generatedChildren" minOccurs="0" maxOccurs="unbounded">
            <xs:complexType>
              <xs:attribute name="ref" type="xs:string" />
            </xs:complexType>
          </xs:element>
        </xs:sequence>
      </xs:extension>
    </xs:complexContent>
  </xs:complexType>
```

*ref* : reserved reference to generated page sets. typical value can be `plugin.goals`

**Comment by bsorrentino** [comment]
This issue is considered a duplicate of issue #124

**Comment by wattazoum** [comment]
Hi, 

this one is meant to have the goal pages generated under a certain page in the page tree, while #124  was to generate the goal table listing in a certain page. 

Today, in the confluence space left menu I have this:
```
site home
| - site home - my subpage 
| - site home - goal1
| - site home - goal2
| - site home - goal3
| - site home - goal4
` - site home - my other subpage 
```
I would like 

```
site home
| - site home - my subpage 
|    | - site home - goal1
|    | - site home - goal2
|    | - site home - goal3
|    ` - site home - goal4
` - site home - my other subpage 
```

**Comment by wattazoum** [comment]
I believe this one is not a duplicate.

**Comment by bsorrentino** [comment]
released with version `5.0-rc3`
